src/metal.cs

```diff
@@ -3684,9 +4045,44 @@ interface MTLTileRenderPipelineColorAttachmentDescriptorArray {
 		void SetObject (MTLTileRenderPipelineColorAttachmentDescriptor attachment, nuint attachmentIndex);
 	}
 
-	[iOS (11, 0), NoTV, NoMac, NoWatch]
+	interface IMTLBinaryArchive {}
+
+	[Mac (11,0), iOS (14,0), TV (14,0)]
+	[Protocol]
+	interface MTLBinaryArchive {
```

Comment:
`diabledefaultctor`?

Reply:
Is a protocol, not an instance of a class, will be converted to an interface, so there is not constructor.